scroll-search.lic

```diff
@@ -0,0 +1,491 @@
+=begin
+  Documentation: TBD
+=end
+
+custom_require.call(%w(common common-items drinfomon))
+
+# Define Scroll Info class
+class ScrollInfo
+  def initialize(container, index, container_index, ordinal, spell, type)
+	# Instance variables
+	@container = container
+	@index = index
+	@container_index = container_index
+	@ordinal = ordinal
+	@spell = spell
+	@type = type
+  end
+
+  attr_accessor :container
+  attr_accessor :spell
+  attr_accessor :ordinal
```

Comment:
Don't feel compelled, but you can list all of these on one line for future information.
`attr_accessor :container, :spell, :ordinal, ...`